tools/autograd/gen_variable_type.py

```diff
@@ -338,6 +338,12 @@
 }
 """)
 
+FW_DERIVATIVE_SETTER_MULTI_OUTPUT = CodeTemplate("""\
+if (${all_res}_new_fw_grad_opt.has_value() && std::get<${idx}>(${all_res}_new_fw_grad_opt.value()).defined()) {
```

Comment:
Is the code sample in the PR description up to date with this? I thing the defined check is missing from there?

Reply:
Updated!